### PR TITLE
Add ability to verify when the last time a client spoke to us

### PR DESCRIPTION
**Pull request by joshua-rule**
Tidies up the subscription code and adds some unit testing.
Primarily adds the ability to check when a client last verified it's still there with a pong response to us pinging it.
Will remove connections/subscriptions that are too old, the client will need to reconnect when it sees we are no longing pinging it.

### CODE REVIEW

File: src/test/java/com/fleetpin/graphql/aws/lambda/admin/AdminTest.java [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        final var admin = createAdmin(databaseManager);
+        createUser(admin);
+
+        {
+            final var current = admin.verify();
+
+            Assertions.assertEquals(1, current.size());
+        }
+
+        Thread.sleep(300);
```
Comment (Aliics) [inline src/test/java/com/fleetpin/graphql/aws/lambda/admin/AdminTest.java]: Would be nice to not use Thread.sleep for less potentially flaky tests.

File: src/main/java/com/fleetpin/graphql/aws/lambda/admin/Admin.java [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+                    .scan(request)
+                    .get()
+                    .items()
+                    .stream()
+                    .map(item -> {
+                        if (verify(item, now)) {
+                            return item;
+                        } else {
+                            return null;
+                        }
```
Comment (Aliics) [inline src/main/java/com/fleetpin/graphql/aws/lambda/admin/Admin.java]: I found this area a little confusing.
Maybe don't have the verify which returns boolean do the disconnecting?

File: src/main/java/com/fleetpin/graphql/aws/lambda/admin/Admin.java [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+            return true;
+        }
+
+        try {
+            disconnect(item.get("connectionId").s());
+        } catch (InterruptedException | ExecutionException e) {
+            throw new RuntimeException(e);
+        }
+
+        return false;
```
Comment (Aliics) [inline src/main/java/com/fleetpin/graphql/aws/lambda/admin/Admin.java]: Could just be a method which returns the condition on 326, if you decide to move the disconnect out of this method.

File: src/main/java/com/fleetpin/graphql/aws/lambda/admin/Admin.java (hunk 2 above)
Comment (joshua-rule) [reply src/main/java/com/fleetpin/graphql/aws/lambda/admin/Admin.java]: yea that makes sense, will change that

**Pull request merged**